match_test.go

```diff
@@ -36,6 +36,31 @@ func TestMatch(t *testing.T) {
 	}
 }
 
+func TestNotMatch(t *testing.T) {
+	var v2 *string
+	tests := []struct {
+		tag   string
+		re    string
+		value interface{}
+		err   string
+	}{
+		{"t1", "[a-z]+", "abc", "must be in a valid format"},
```

Comment:
So we only get "must be in a valid format" as error message, nothing else to precise we are in an "inverted mode" ?

Reply:
I think it's simpler to use the same generic sentinel error in this case. And since we override most _(if not all)_ of our validation error messages: I don't think that's an issue.
However, I have a low opinion on this. If you have something in mind, don't hesitate to contribute directly in this branch, or another.

Reply:
> you mean in Koyeb apps ?

Yes :slightly_smiling_face: